src/DynamoPackages/PackageLoader.cs

```diff
@@ -503,7 +500,18 @@ private void ScanPackageDirectories(string root, IPreferences preferences)
 
                     var pkg = ScanPackageDirectory(dir, checkCertificates);
                     if (pkg != null && preferences.PackageDirectoriesToUninstall.Contains(dir))
-                        pkg.MarkForUninstall(preferences);
+                    {
+                        if (pkg.BuiltInPackage)
+                        {
+                            // If the discovered package is built-in and marked to be unloaded then we set it directly to Unloaded state.
```

Comment:
so this comment says unloaded - but it's really if the built-in package's package root dir was contained in the uninstall list in preferences - yes?

Reply:
Correct. I can replace it (makes it clearer)